src/signals/IncidentMap/components/Header/styled.tsx

```diff
@@ -28,8 +28,15 @@ export const Title = styled.div`
   height: 100%;
   margin-left: ${themeSpacing(4)};
 
-  img {
+  img,
+  a {
+    display: block;
+    width: auto;
     height: 100%;
-    max-width: 90px;
+    max-height: ${themeSpacing(HEADER_HEIGHT_DESKTOP - 5)};
+
+    @media screen and (${breakpoint('max-width', 'tabletM')}) {
+      max-height: ${themeSpacing(HEADER_HEIGHT_MOBILE - 5)};
+    }
```

Comment:
for some reason an earlier comment didn't make it.

I would reverse these max height declarations and make it a  min-height instead. => mobile first